common/hooks/do-extract/00-distfiles.sh

```diff
@@ -163,4 +152,27 @@ hook() {
 			;;
 		esac
 	done
+
+	# find "$extractdir" -mindepth 1 -maxdepth 1 -printf '1\n' | wc -l
```

Comment:
could this be done with `ls -A1 | wc -l`?

Reply:
Thinking again, I think I prefer a shell for of globs instead, `ls | wc` needs 2 `fork`s and `pipe`.

Reply:
And we need to get the subdirs' name anyway :-p